public/docs/content/getting_started/.pages.yml

```diff
@@ -0,0 +1,8 @@
+---
+  nav:
+    - Getting started: index.md
```

Comment:
This is empty and can be removed.

Reply:
comment: I'd like to add summary content here